docs/learning/Bottlenecking.md

```diff
@@ -1,37 +1,86 @@
+
+AMD, Nvidia and Intel all have first party monitoring software you can use with their products; [AMD Adrenalin](https://www.amd.com/en/products/software/adrenalin.html), [Nvidia Geforce Experience](https://www.nvidia.com/en-us/geforce/geforce-experience/), and [Intel Arc Control Center](https://www.intel.com/content/www/us/en/products/docs/discrete-gpus/arc/software/arc-control.html); you should pick the one that matches your GPU.
+
+To see if you have a GPU bottleneck you play a game with the monitoring software overlay on the screen and watch your GPU utilization percentage. If the GPU is staying at or near 100% utilization in various scenes, then you have a GPU bottleneck in that game. If you want higher FPS you can turn down the graphical settings, making it easier for your GPU to draw each frame and your FPS should go up. Different settings can have a more noticeable impact on performance or on visual quality than others.
+
+To see if you have a CPU bottleneck you'd do the same process and look at your GPU utilization percentage, the farther away from 100% it's staying, the more headroom your current GPU has. You can try turning up some graphical settings to improve visual quality. With most settings, this will add more work for the GPU while adding little to no extra work for the CPU. Ray-tracing and Draw Distance are two settings that typically do increase the amount of work the CPU has to do. It's best to experiment a bit and find what looks and feels the best to you.
+
+If you have a severe bottleneck and want more performance out of your PC but adjusting the balance of the settings isn't improving things enough for you then it would be time to consider hardware upgrades.
+
+## Why am I not looking at CPU utilization?
+
+Most games are *not designed to utilize 100% of a CPU*, especially with newer CPUs having higher core and thread counts. Many games care more about each core's performance than they do about how many cores you have.
+
+For example: a 6-core AMD 5600X or an 8-core AMD 5800X will perform about as well in most games as a 16-core AMD 5950X. The 5600X might show fairly high utilization while the 5950X shows very low utilization because a specific game was only designed to really utilize 4 cores, or even only 1 core.
+
+## What about RAM?
+
+For the most part RAM is *tied* into CPU performance.
+
+There are three aspects to ram performance: capacity (how many GB of ram you have), bandwidth (what frequency your ram is running at and whether it's in dual channel or not), and latency (what timings your RAM is running at, notably different from frequency).
+
+The main one that can hold you back is *capacity*. If your computer runs out of free RAM space, your CPU will have to ask for information from the storage drive directly which takes significantly longer than having things stored in RAM and ready for the CPU to see. The best thing to do would be to upgrade or add more RAM.

```

Comment:
I would link somewhere in here our guide for needing more ram.

Reply:
Resolved as of commit [5000c0e](https://github.com/r-Techsupport/rTS_Wiki/pull/298/commits/5000c0e3ff69c9bfb9426b73a699bd59dbdfffa7).